Source/WebCore/platform/graphics/Font.h

```diff
@@ -149,7 +149,7 @@ class Font : public RefCounted<Font> {
     };
     float widthForGlyph(Glyph, SyntheticBoldInclusion = SyntheticBoldInclusion::Incorporate) const;
 
-    const Path& pathForGlyph(Glyph) const; // Don't store the result of this! The hash map is free to rehash at any point, leaving this reference dangling.
+    Path pathForGlyph(Glyph) const;
```

Comment:
Do we need to do something at the call site to make the caller not perform a second copy?

Reply:
I think the return value optimization takes care of this.